features/bevragen/persoon/verblijfplaats/fields-alias-fout-cases.feature

```diff
@@ -10,16 +12,13 @@ Functionaliteit: verblijfplaats - fields alias fout cases
     | burgerservicenummer | 000000097                       |
     | fields              | <field>                         |
     Dan heeft de response een object met de volgende gegevens
-    | naam     | waarde                                                      |
-    | type     | https://datatracker.ietf.org/doc/html/rfc7231#section-6.5.3 |
-    | title    | U bent niet geautoriseerd voor deze operatie.               |
-    | status   | 403                                                         |
-    | detail   | De foutieve parameter(s) zijn: fields.                      |
-    | code     | autorisation                                                |
-    | instance | /haalcentraal/api/brp/personen                              |
-    En heeft het object de volgende 'invalidParams' gegevens
-    | code         | name   | reason                                                           |
-    | autorisation | fields | niet geautoriseerd voor één of meerdere opgegeven field waarden. |
+    | naam     | waarde                                                                  |
+    | type     | https://datatracker.ietf.org/doc/html/rfc7231#section-6.5.3             |
+    | title    | U bent niet geautoriseerd voor één of meerdere opgegeven field waarden. |
+    | status   | 403                                                                     |
+    | detail   | De foutieve field waarden zijn: fields[0].                              |
+    | code     | authorization                                                           |
+    | instance | /haalcentraal/api/brp/personen                                          |
 
     Voorbeelden:
```

Comment:
De onderstaande voorbeelden impliceren de volgende Rule:

`de field alias 'verblijfplaatsBinnenland' moet worden gebruikt door een consumer die niet geautoriseerd is voor het vragen van 'verblijfplaats buitenland' velden'`

want als dat niet waar is dan zouden de voorbeelden in regel 25 t/m 27 niet persé tot een fout hoeven te leiden.

of de omgekeerde variant daarvan:

`de standaard verblijfplaats field paden mag niet worden gebruikt door een consumer die niet is geautoriseerd voor het bevragen van 'verblijfplaats buitenland' velden`